requirements/data/base.txt

```diff
@@ -0,0 +1,12 @@
+# NOTE: the upper bound for the package version is only set for CI stability, and it is dropped while installing this package
```

Comment:
the same for all requirements files in this folder, are they needed, I do not see their usage...